### ISSUE TITLE
Send screen in a Watch-only wallet doesn't show Allow Fee Bump. Also the PSBT generated have RBF == FALSE

### ISSUE THREAD

**Issue by tadeubas** [BlueWallet/BlueWallet#5245]
Send screen in a Watch-only wallet doesn't show `Allow Fee Bump` option within the 3-dot (`...`) menu. Also, the PSBT is generated with RBF == FALSE which is opposite behavior for BW, which defaults to RBF == TRUE

**Comment by Overtorment** [comment]
yes, multisig wallets and watch-only wallets (with hw wallets integration) dont yet support fee bump

**Comment by Cindysexy** [comment]
I don't understand about how to get my money

**Comment by tiagodread** [comment]
+1 for this feature
